### DISCUSSION TITLE
How to unselect in the CopyMode

### DISCUSSION THREAD

**Discussion by zenithyr** [wezterm/wezterm#2246] [Q&A] [answered]
First of all, let me say how awesome the tool is. Can't believe I can now throw away tmux + alacritty/kitty/iTerm2 completely while enjoying the performance and level of customization I've enjoyed from the tool.

I have one minor complaint regarding the CopyMode. After I activate the CopyMode, I can use ```v``` to start mark selection. But **if I regret and want to discard the selection completely**, it seems impossible other than re-activate the mode, losing all the progress if I have scrolled a few pages back.

I tried [ClearSelection](https://github.com/wez/wezterm/issues/1900) but it just clears the mark on the screen, the next moment if I type ```v``` again the previous selected area is still "memorized".

I also tried to map ```act.Multiple(act.CopyMode("close"), act.ActivateCopyMode)``` with no luck, and it seems a bit hacky.

Can anyone share some thoughts? Thanks.

**Comment by wez** [comment, marked as answer]
In 10121790df10fbc40896f567046b6fcd54c42bfb, the behavior is now more like vim: press `v` to start selecting, then press it again to stop and clear the selection.

It typically takes about an hour before fixes are available as nightly builds for all platforms.  Linux builds are the fastest to build and are often available within about 20 minutes.  Windows and macOS builds take a bit longer.

Please take a few moments to try out the fix and let me know how that works out. You can find the nightly downloads for your system in the [wezterm installation docs](https://wezfurlong.org/wezterm/installation.html).

If you prefer to use packages provided by your distribution or package manager of choice and don't want to replace that with a nightly download, keep in mind that you can download portable packages (eg: a `.dmg` file on macOS, a `.zip` file on Windows and an `.AppImage` file on Linux) that can be run without permanently installing or replacing an existing package, and can then simply be deleted once you no longer need them. 

If you are eager and can [build from source](https://wezfurlong.org/wezterm/install/source.html) then you may be able to try this out more quickly.

**Reply by zenithyr** [reply]
Thanks for the quick turnaround!
